cores/arduino/stm32/uart.c

```diff
@@ -732,6 +742,34 @@ void uart_attach_tx_callback(serial_t *obj, int (*callback)(serial_t *))
   HAL_NVIC_EnableIRQ(obj->irq);
 }
 
+/**
+ * Enable transmitter for half-duplex mode. NOOP in full-fuplex mode
+ * 
+ * @param obj : pointer to serial_t structure
+ * @retval none
+ */
+void uart_enable_tx(serial_t *obj)
+{
+  if (obj == NULL || obj->pin_rx != NC) {
```

Comment:
```suggestion
  if ((obj != NULL) && (obj->pin_rx == NC)) {
```

invert test and avoid using `return` (I know this is done in other function but I try to remove them when I modify the code :wink: ).

Reply:
done